python/feathub/examples/feathub_flink_cli.py

```diff
@@ -0,0 +1,90 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from datetime import timedelta
+
+from feathub.common.types import Int32, Timestamp, Float64
+from feathub.feathub_client import FeathubClient
+from feathub.feature_views.derived_feature_view import DerivedFeatureView
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.transforms.over_window_transform import OverWindowTransform
+from feathub.processors.flink.flink_feature_table import (
+    FlinkConnectorSource,
+    FlinkConnectorSink,
+)
+from feathub.table.schema import Schema
+
+
+def main() -> None:
+    client = FeathubClient(
+        config={
+            "processor": {
+                "processor_type": "flink",
+                "flink": {"deployment_mode": "cli"},
+            },
+            "online_store": {
+                "memory": {},
+            },
+            "registry": {
+                "registry_type": "local",
+                "local": {
+                    "namespace": "default",
+                },
+            },
+            "feature_service": {
+                "service_type": "local",
+                "local": {},
+            },
+        }
+    )
+
+    source = FlinkConnectorSource(
```

Comment:
Does this mean that some source/slink in FeatHub can only be interpreted by Flink processor? This will break the current FeatHub design where user's FeatHub program (e.g. their source/sink definition) can be interpreted by any processor (e.g. local processor, Flink processor).

Can we add classes `DataGenSource` and `PrintSink` and let users use these classes directly?